### PR TITLE
Add WG-Allocators team

### PR DESCRIPTION
**Pull request by TimDiekmann**
cc @scottjmaddox If you want me to add you to the mailing list, I need your e-mail address.

### CODE REVIEW

**Comment by pietroalbini** [comment]
Looks good! I'll merge once I get an approval from @rust-lang/libs

**Comment by lqd** [comment]
if this is a t-libs WG, maybe the toml file should mention `subteam-of = "libs"` ?

**Comment by scottjmaddox** [comment]
> cc @scottjmaddox If you want me to add you to the mailing list, I need your e-mail address.

@TimDiekmann Please add me as my github username at gmail. Thanks!

File: teams/wg-allocators.toml [hunk 1]
```diff
@@ -21,3 +21,6 @@ name = "Allocator working group"
 description = "Paving a path for a standard set of allocator traits to be used in collections."
 repo = "https://github.com/rust-lang/wg-allocators"
 zulip-stream = "t-libs/wg-allocators"
+
+[[lists]]
+address = "allocators@rust-lang.org"
```
Comment (pietroalbini) [inline teams/wg-allocators.toml:26]:
```suggestion
address = "wg-allocators@rust-lang.org"
```

File: teams/wg-allocators.toml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "Amanieu",
+    "CAD97",
+    "Lokathor",
+    "scottjmaddox",
+    "TimDiekmann",
+    "Wodann"
+]
+
+[website]
+page = "allocators"
```
Comment (lqd) [inline teams/wg-allocators.toml:19]: @pietroalbini this `page` key is also for top-level teams, right ?

Comment (pietroalbini) [reply teams/wg-allocators.toml:19]: That shouldn't be needed here, the `page` key exists as a workaround for other teams.

**Comment by BurntSushi** [comment]
LGTM

**Pull request merged**